crypto/test/key-compression.ts

```diff
@@ -15,7 +15,7 @@ test('public key compression', async (t) => {
 
 test('works consistently', async (t) => {
   const pubkeys = []
-  for (let i = 0; i < 100; i++) {
+  for (let i = 0; i < 10000; i++) {
```

Comment:
is this secretly a token miner

Reply:
hehe 

it probably doesn't _need_ to be that high. I did have an error in it from not left padding the y coord which would only pop up once every ~256 keys. So I figured why not throw 10000 at it to make sure 😅 
I'll drop it down to just 1000 for now